quirks/password-rules.json

```diff
@@ -779,6 +779,9 @@
     "prestocard.ca": {
         "password-rules": "minlength: 8; required: lower; required: upper; required: digit,[!\"#$%&'()*+,<>?@];"
     },
+    "pret.com": {
+        "password-rules": "minlength: 12; required: lower; required: digit; required: [@$!%*#?&]; allowed: lower;"
```

Comment:
Do you need both `required: lower` and `allowed: lower`?

Reply:
I don’t! I actually intended for the allowed to be for upper!